policy/scan/local_scanner.go

```diff
@@ -523,54 +528,9 @@ func (s *LocalScanner) distributeJob(job *Job, ctx context.Context, upstream *up
 	return reporter.Reports(), finished, nil
 }
 
-func (s *LocalScanner) upstreamServices(conf *upstream.UpstreamConfig) *policy.Services {
-	if conf == nil ||
-		conf.ApiEndpoint == "" ||
-		conf.Incognito {
-		return nil
-	}
-
-	client, err := s.upstreamClient(conf)
-	if err != nil {
-		log.Error().Err(err).Msg("could not init upstream client")
-		return nil
-	}
-
-	res, err := policy.NewRemoteServices(client.ApiEndpoint, client.Plugins, client.HttpClient)
-	if err != nil {
-		log.Error().Err(err).Msg("could not connect to upstream")
-	}
-
-	return res
-}
-
 func (s *LocalScanner) RunAssetJob(job *AssetJob) {
 	log.Debug().Msgf("connecting to asset %s", job.Asset.HumanName())
 
-	upstream := s.upstreamServices(job.UpstreamConfig)
-	if upstream != nil {
-		resp, err := upstream.SynchronizeAssets(job.Ctx, &policy.SynchronizeAssetsReq{
-			SpaceMrn: job.UpstreamConfig.SpaceMrn,
-			List:     []*inventory.Asset{job.Asset},
-		})
-		if err != nil {
-			log.Error().Err(err).Msgf("failed to synchronize asset to Mondoo Platform %s", job.Asset.Mrn)
```

Comment:
That's the only difference I can spot.

When a single asset cannot be synced, we do not add an error to the progress bar, but ...
Actually not sure, what happens. I'd like to create a testcase for this.

Reply:
I took the registry scan to create an inventory and changed one asset to not be reachable.

The behavior between the current cnspec and this PR is the same. Three assets get synchronized upstream.